Oh interesting, this fails to build:
```
In file included from /Volumes/Data/worker/macOS-Safer-CPP-Checks-EWS/build/Source/WTF/wtf/StdLibExtras.h:44:
/Volumes/Data/worker/macOS-Safer-CPP-Checks-EWS/build/Source/WTF/wtf/Assertions.h:983:24: error: 'always_inline' attribute ignored [-Werror,-Wignored-attributes]
  983 | NO_RETURN_DUE_TO_CRASH ALWAYS_INLINE void WTFCrashWithInfo(int line, const char* file, const char* function);
      |                        ^
In file included from /Volumes/Data/worker/macOS-Safer-CPP-Checks-EWS/build/Source/WTF/wtf/WTFPrefix.h:31:
In file included from /Volumes/Data/worker/macOS-Safer-CPP-Checks-EWS/build/Source/WTF/wtf/HashSet.h:24:
/Volumes/Data/worker/macOS-Safer-CPP-Checks-EWS/build/Source/WTF/wtf/Compiler.h:197:45: note: expanded from macro 'ALWAYS_INLINE'
  197 | #define ALWAYS_INLINE inline __attribute__((__always_inline__))
      |                                             ^
In file included from /Volumes/Data/worker/macOS-Safer-CPP-Checks-EWS/build/Source/WTF/wtf/WTFPrefix.h:31:
In file included from /Volumes/Data/worker/macOS-Safer-CPP-Checks-EWS/build/Source/WTF/wtf/HashSet.h:27:
In file included from /Volumes/Data/worker/macOS-Safer-CPP-Checks-EWS/build/Source/WTF/wtf/HashTable.h:32:
In file included from /Volumes/Data/worker/macOS-Safer-CPP-Checks-EWS/build/Source/WTF/wtf/AlignedStorage.h:29:
In file included from /Volumes/Data/worker/macOS-Safer-CPP-Checks-EWS/build/Source/WTF/wtf/StdLibExtras.h:44:
/Volumes/Data/worker/macOS-Safer-CPP-Checks-EWS/build/Source/WTF/wtf/Assertions.h:1054:20: note: conflicting attribute is here
 1054 |     __attribute__((optnone))
      |                    ^
1 error generated.
Command ProcessPCH++ failed with a nonzero exit code
```

--- a/Source/WTF/wtf/Assertions.h
+++ b/Source/WTF/wtf/Assertions.h
@@ -979,7 +979,7 @@ WTF_EXPORT_PRIVATE NO_RETURN_DUE_TO_CRASH NOT_TAIL_CALLED void WTFCrashWithInfoI
 WTF_EXPORT_PRIVATE NO_RETURN_DUE_TO_CRASH NOT_TAIL_CALLED void WTFCrashWithInfoImpl(int line, const char* file, const char* function, UCPURegister reason, UCPURegister misc1, UCPURegister misc2);
 WTF_EXPORT_PRIVATE NO_RETURN_DUE_TO_CRASH NOT_TAIL_CALLED void WTFCrashWithInfoImpl(int line, const char* file, const char* function, UCPURegister reason, UCPURegister misc1);
 WTF_EXPORT_PRIVATE NO_RETURN_DUE_TO_CRASH NOT_TAIL_CALLED void WTFCrashWithInfoImpl(int line, const char* file, const char* function, UCPURegister reason);
-#if !ASAN_ENABLED && (OS(DARWIN) || PLATFORM(PLAYSTATION)) && (CPU(X86_64) || CPU(ARM64))
+#if !ASAN_ENABLED && !defined(__clang_analyzer__) && (OS(DARWIN) || PLATFORM(PLAYSTATION)) && (CPU(X86_64) || CPU(ARM64))
 NO_RETURN_DUE_TO_CRASH ALWAYS_INLINE void WTFCrashWithInfo(int line, const char* file, const char* function);
 #else
 NO_RETURN_DUE_TO_CRASH NOT_TAIL_CALLED void WTFCrashWithInfo(int line, const char* file, const char* function);
@@ -1033,7 +1033,7 @@ NO_RETURN_DUE_TO_CRASH ALWAYS_INLINE void WTFCrashWithInfo(int line, const char*
     WTFCrashWithInfoImpl(line, file, function, wtfCrashArg(reason), wtfCrashArg(misc1), wtfCrashArg(misc2), wtfCrashArg(misc3), wtfCrashArg(misc4), wtfCrashArg(misc5), wtfCrashArg(misc6));
 }
 
-#if !ASAN_ENABLED && (OS(DARWIN) || PLATFORM(PLAYSTATION)) && (CPU(X86_64) || CPU(ARM64))
+#if !ASAN_ENABLED && !defined(__clang_analyzer__) && (OS(DARWIN) || PLATFORM(PLAYSTATION)) && (CPU(X86_64) || CPU(ARM64))
 
 NO_RETURN_DUE_TO_CRASH ALWAYS_INLINE void WTFCrashWithInfo(int line, const char* file, const char* function)
 {

